scripts-dev/check_line_terminators.sh

```diff
@@ -28,4 +28,7 @@
 cd `dirname $0`/..
 
 # Find and print files with non-unix line terminators
-find . -path './.git/*' -prune -o -type f -print0 | xargs -0 grep -I -l $'\r$' && ( echo 'found files with CRLF line endings'; exit 1 )
+if find . -path './.git/*' -prune -o -type f -print0 | xargs -0 grep -I -l $'\r$'; then
+    echo -e '\e[31mERROR: found files with CRLF line endings. See above.\e[39m'
```

Comment:
decided to make it red while I was here.

Reply:
The red text didn't work for me when I ran this as `./scripts-dev/check_line_terminators.sh`

(Directly running this line did work though...)

Not sure if it is mac / zsh messing something up though.

Reply:
odd. It works here, and I'm pretty confident it will work on buildkite (we have other scripts which do the same). How sad does it make you that it doesn't work for you?

Reply:
Not sad enough to block merging this. 👍